sql/rpl_info_table.cc

```diff
@@ -21,6 +21,46 @@
 #include "rpl_info_values.h"        // Rpl_info_values
```

Comment:
But how this test will fail in the case of regression? There are no strict conditions of failure. Ok, we will have a little bit better code coverage, but coverage itself is useless until we have criteria of failure or correctness, what is not true for this test case.